README.md

```diff
@@ -327,13 +336,14 @@ There is also a [FAQ](https://github.com/goadesign/goa/blob/v2/docs/FAQ.md) and
 a document describing
 [error handling](https://github.com/goadesign/goa/blob/v2/docs/ErrorHandling.md).
 
+If you are coming from v1 you may also want to read the 
+[Upgrading](https://github.com/goadesign/goa/blob/v2/docs/Upgrading.md) document.
+
 ### Examples
 
-The [examples](https://github.com/goadesign/goa/tree/v2/examples) directory
+The [examples](https://github.com/goadesign/goa/tree/master/examples) directory
```

Comment:
Didn't we move examples to its own repository?

Reply:
Thanks, fixed.